service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/SubmittedCaseEventHandler.java

```diff
@@ -56,7 +60,9 @@ public void notifyAdmin(final SubmittedCaseEvent event) {
         NotifyData notifyData = hmctsEmailContentProvider.buildHmctsSubmissionNotification(caseData);
         String recipient = adminNotificationHandler.getHmctsAdminEmail(caseData);
 
-        notificationService.sendEmail(HMCTS_COURT_SUBMISSION_TEMPLATE, recipient, notifyData, caseData.getId());
+        String template = toggleService.isEldestChildLastNameEnabled() ? HMCTS_COURT_SUBMISSION_TEMPLATE_CHILD_NAME
```

Comment:
can remove duplication of this code

Reply:
extracted to method that accepts the template strings as parameters